lms/templates/header.html

```diff
@@ -0,0 +1,19 @@
+<%namespace name='static' file='static_content.html'/>
+<%!
+from microsite_configuration import microsite
+%>
+<%
+theme_enabled = settings.FEATURES.get("USE_CUSTOM_THEME", False)
```

Comment:
I can't tell if you are asking an informational question, or asking for a change to this line of code.